docs/source/advanced.rst

```diff
@@ -124,6 +124,20 @@ Limitations
 
 - Passing the Profiler name as a string "hpu" to the trainer is not supported.
 
+----
+
+Using Intel Gaudi Profiler
+------------------------------
+
+The Intel Gaudi Profiling subsystem, and the Profiling Configuration tools are methods to configure Intel Gaudi Profiler.
+
+Please refer to `Getting Started with Intel Gaudi Profiler <https://docs.habana.ai/en/latest/Profiling/Intel_Gaudi_Profiling/Getting_Started_with_Profiler.html>`_ for more information.
+
+.. note::
+
+    ``HPUProfiler`` and Intel Gaudi Profiler should not be used together. Therefore, `HABANA_PROFILE` should not be set in environment when using ``HPUProfiler``.
```

Comment:
will hpuprofiler work now ?

Reply:
yes.  This value is set to "profile_api_light" internally to enable profiler.